kotlinx-coroutines-core/common/src/flow/terminal/Reduce.kt

```diff
@@ -138,10 +138,10 @@ public suspend fun <T : Any> Flow<T>.firstOrNull(): T? {
 }
 
 /**
- *  The terminal operator that returns the first element emitted by the flow matching the given [predicate] and then cancels flow's collection.
+ * The terminal operator that returns the first element emitted by the flow matching the given [predicate] and then cancels flow's collection.
  * Returns `null` if the flow did not contain an element matching the [predicate].
```

Comment:
Do we want to comment on what happens if the flow contains a single `null` value or let that be implied?